docs/src/release_notes/v0.26.x.md

```diff
@@ -0,0 +1,15 @@
+## v0.26.x (upcoming release)
```

Comment:
Another important change is missing:

* Required custom Mitsuba build bumped to v0.2.0 (based on Mitsuba v3.4.1)